gallery/src/main.scss

```diff
@@ -110,3 +110,28 @@
     display: none;
   }
 }
+
+// Accessibility Checklist Page
+.nx-list.nx-list--no-bullets {
+  .nx-list__item {
+    list-style: none;
+  }
+}
+
+.nx-tile--accessibility-list {
+  .nx-radio-checkbox__content {
+    overflow: unset;
+    text-overflow: unset;
+    white-space: normal;
```

Comment:
The only one of these that's really necessary to reset is `white-space`

Reply:
> The only one of these that's really necessary to reset is `white-space`

https://github.com/sonatype/sonatype-react-shared-components/pull/657/commits/d5e3865a0eecfe3d0e95efc3c55868d5afc1c35e